BaragonService/src/main/java/com/hubspot/baragon/service/config/PurgeCacheConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package com.hubspot.baragon.service.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.hubspot.baragon.models.BaragonService;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PurgeCacheConfiguration {
+  @JsonProperty("enabledTemplates")
+  @NotNull
+  private List<String> enabledTemplates;
+
+  @JsonProperty(value = "excludedServiceIds")
+  @NotNull
+  private List<String> excludedServiceIds;
+
+  public List<String> getEnabledTemplates() {
+    if (enabledTemplates == null) {
+      return new ArrayList<>();
+    }
+    return enabledTemplates;
+  }
+
+  public void setEnabledTemplates(List<String> enabledTemplates) {
+    this.enabledTemplates = enabledTemplates;
+  }
+
+  public List<String> getExcludedServiceIds() {
+    if (excludedServiceIds == null) {
+      return new ArrayList<>();
+    }
+    return excludedServiceIds;
+  }
+
+  public void setExcludedServiceIds(List<String> excludedServiceIds) {
+    this.excludedServiceIds = excludedServiceIds;
+  }
+
+  public boolean serviceShouldPurgeCache(BaragonService service){
```

Comment:
These config java classes are more about the representation of the config yaml in java. Any logic like this is better suited for another class since it isn't part of the config yaml structure